dependencies_2.12.lock

```diff
@@ -5800,5 +5800,52 @@
             "locked": "1.3.6",
             "requested": "1.3.6"
         }
+    },
+    "zinc": {
```

Comment:
hmm where is this coming from? We're pulling in scala 2.10 as well

@tomnis

Reply:
hmm its in the zinc configuration... this has to do with gradle scala plugin resolving scalac. i think this is ok. scalac itself needs a scala-library that may be different from what our library is using
https://docs.gradle.org/current/userguide/scala_plugin.html#sec:configure_zinc_compiler

however, dependency lock files should be updated in lock-step (the 2.11 file as well).

@el5846 can you back out this change to 2.12 lockfile?